AltBeacon.Library/src/AltBeacon/Beacon/BeaconParser.cs

```diff
@@ -416,8 +417,7 @@ public BeaconParser SetBeaconLayout(string beaconLayout)
 
                 if (!found)
                 {
-                    ////TODO LogManager
-                    ////LogManager.d(Tag, "cannot parse term " + term);
+                    Logger.Debug("cannot parse term " + term);
```

Comment:
No, I don't think that is necessary.  If you have VerboseLoggingEnabled set to false, lines like `Logger.Debug("cannot parse term " + term);` will do nothing anyway.  All you are accomplishing by wrapping these in if statements like:

    if (LogManager.VerboseLoggingEnabled) {
      Logger.Debug("cannot parse term " + term);
    }

is making it so the string object construction and concatenation does not take place unnecessarily, which saves processing time.  Even if you don't do this everywhere,  doing it inside beacon parsing is important because it is gets executed in a tight loop.